packages/tupaia-web/src/theme/theme.ts

```diff
@@ -0,0 +1,39 @@
+import { createMuiTheme } from '@material-ui/core';
+
+export const theme = createMuiTheme(
+  {
+    palette: {
+      type: 'dark',
+      primary: {
+        main: '#1978D4', // Main blue (as seen on primary buttons)
+      },
+      secondary: {
+        main: '#ee6230', // Tupaia Orange
+      },
+      background: {
+        default: '#262834', // Dark blue background
+      },
+    },
+    typography: {
+      h2: {
+        fontSize: '1.2rem',
+      },
+    },
+  },
+  {
+    topBarHeight: {
+      default: 60,
+      mobile: 50,
+    },
+    mapOverlaySelector: {
+      width: 340,
+      background: 'rgb(7, 40, 73)',
+      datePicker: '#0F2847',
+      library: {
+        expanded: '#EFEFF0',
+        collapsed: 'rgba(255, 255, 255, 0.6)',
+      },
+      divider: '#203e5c',
+    },
```

Comment:
```suggestion
   
```
I don't think that these are theme specific variables so they could go in constants or in the case of the topBarHeight just go in the TopBar Component file.